recipe/meta.yaml

```diff
@@ -641,35 +643,19 @@ outputs:
         - {{ pin_subpackage('libopenvino-tensorflow-frontend', max_pin='x.x.x') }}
         - {{ pin_subpackage('libopenvino-tensorflow-lite-frontend', max_pin='x.x.x') }}
     requirements:
-      host:
-        - python # dummy dependency to create python-version dependent package
       run:
-        - python # dummy dependency to create python-version dependent package
         - {{ pin_subpackage('libopenvino-dev', exact=True) }}
-        - {{ pin_subpackage('libopenvino-python', exact=True) }}
+        # Do not use the exact pin here since we are generating multiple different 
+        # python packages in 1 go. Instead let the common python constraints
+        # choose the same variant of the package
+        # - {{ pin_subpackage('libopenvino-python', exact=True) }}
+        - {{ pin_subpackage('libopenvino-python', max_pin='x.x.x') }}
     test:
-      requires:
-        - {{ compiler('cxx') }}
-        - {{ compiler('c') }}
-        - ninja
-        - pkg-config
-        - cmake
-        - sysroot_linux-64 =2.17  # [linux64]
       files:
         - snippets/
       commands:
         # check that devices and frontends are found using Python API
-        - $PYTHON snippets/openvino_available_components.py  # [unix]
-
-        # check that devices are found using OpenVINO C API
-        - $CC snippets/openvino_available_devices.c $(pkg-config --cflags --libs openvino) -Wl,-rpath,$PREFIX/lib -o openvino_devices_test  # [unix]
-        - ./openvino_devices_test  # [unix]
-
-        # check that frontends are found using OpenVINO C++ API
-        - cmake -G "Ninja Multi-Config" -S snippets -B snippets_openvino_{{ CONDA_PY }}_build
-        - cmake --build snippets_openvino_{{ CONDA_PY }}_build --config Release --verbose
-        - ./snippets_openvino_{{ CONDA_PY }}_build/Release/openvino_frontends_test  # [unix]
-        - .\\snippets_openvino_{{ CONDA_PY }}_build\\Release\\openvino_frontends_test.exe  # [win]
```

Comment:
These seems seem to be duplicated from `libopenvino-dev` lets just see if removing them reduces the build time.